backend/api/serializers/GroupSerializer.py

```diff
@@ -0,0 +1,44 @@
+from api.models import Group
+from rest_framework import serializers
+from api.CustomField import BlankableIntegerField
+
+
+class GroupSerializer(serializers.ModelSerializer):
+    name = serializers.CharField(read_only=True)
+    min_age = BlankableIntegerField(required=False)
+    max_age = BlankableIntegerField(required=False)
+
+    class Meta:
+        model = Group
+        fields = ('id', 'name', 'gender', 'min_age', 'max_age')
+
+    def validate(self, data):
+        data = super().validate(data)
+        data = self.validate_age(data)
+        data = self.validate_unique_group(data)
+
+        return data
+
+    def validate_age(self, data):
```

Comment:
It would be good to consider the following scenarios when validating the ages, right now it's allowing me to create a group "Boys11to0" when providing min_age=11 and max_age=0


{   "id": 8,   "name": "Boys11to0",   "gender": "M",   "min_age": 11,   "max_age": 0 }

Also, with these values:

{
  "gender": "M",
  "min_age": 11,
  "max_age": -3
}
is throwing this error: "Minimum age cannot be greater or equal to maximum age"

Reply:
I added a min value validator to both `min_age` and `max_age` on the serializer.

Adding the validator on the model does not work because we are using a custom field on the serializer.

Now we get the appropriate error:

<img width="207" alt="image" src="https://github.com/MarisiaS/SMM/assets/95152310/6512fdf7-9807-45c3-a496-8c3d3d762052">


<img width="204" alt="image" src="https://github.com/MarisiaS/SMM/assets/95152310/49d719da-afed-4c06-a985-139048cf1839">